horde/classes/base/waiting_prompt.py

```diff
@@ -131,6 +131,12 @@ def activate(self):
             self.extra_priority = round(self.user.kudos / 100) 
         else:    
             self.extra_priority = self.user.kudos
+        # This is an extra cost for the operation as a whole, to represent the infrastructure costs
+        # and rewarding requests which bundle multiple jobs into the same payload
+        # Instead of splitting them into multiples.
+        horde_tax = 2

```

Comment:
It is my view that this change will only be punitive to kudos holders, by definition, and will not disincentivize front-ends from continuing to not batch requests (with params like "n"). Anonymous users are not impacted, and as such, the only downside ends up to being passed on to kudos holders who do not have any say in how the frontends decide to structure multiple requests.